app/models/game.rb

```diff
@@ -1,6 +1,67 @@
 class Game < ApplicationRecord
   has_many :user_games
   has_many :users, through: :user_games
+  has_many :game_pieces
+  has_many :pieces, through: :game_pieces
+
+  after_create :lay_out_board!
+
+  def lay_out_board!
```

Comment:
would it make sense to use i here? I may be completely off, but I used it with my mentor recently with something else and wondered if it would fit here

Reply:
Hi @tscheitle, can you elaborate a bit on what you mean by |i|?